tests/lib/nested.sh

```diff
@@ -1343,6 +1343,121 @@ nested_start_core_vm_unit() {
     fi
 }
 
+
+nested_start_lxd_vm() {
+    local QEMU CURRENT_IMAGE
+    CURRENT_IMAGE=$1
+
+    snap install lxd
+    lxd init --auto
+
+    wget -q https://github.com/canonical/lxd/releases/latest/download/bin.linux.lxd-migrate.x86_64
```

Comment:
do you know why the tool isn't distributed within the snap?

Reply:
Because its for migrating machines into an existing LXD installation. Rather than installing on your LXD host.

https://documentation.ubuntu.com/lxd/en/latest/howto/import_machines_to_instances/